.testcaferc.js

```diff
@@ -82,6 +82,10 @@ const config = {
     // only check one of {gen3 | gen2} conditionals. without this guard, a
     // fixture or test will always get skipped in both testcafe runs
     if (env.OKTA_SIW_GEN3) {
+      // Do not execute Gen 1 tests in parity suite
+      if (fixtureMeta.gen1 === true) {
+        return false;
+      }
```

Comment:
Would it be better to use the path of the spec file to filter in this case since they all are under `/spec/v1`? Hopefully your change to add `.meta('gen1` is in a single commit if we do this. I also am not sure if there's any harm in adding the meta values anyways so just a thought.

Reply:
I initially tried using the fixturePath param, and I did `fixturePath.endsWith('v1') but that did not work, it must end with something else (I didnt debug more) so I just tried the meta flag and that immediately worked. But I can look more into why the fixture path didnt work.

Reply:
FYI, updated this to use the fixture path, I ended up using `includes` instead of endsWith